src/app/Main/Body/QueryPage/Results/Charts/QueryPageContent-BarChart.component.tsx

```diff
@@ -21,11 +21,12 @@ import { BarChartResultsComponent } from "./BarChartResults.component";
 
 const dataStyle = css`
     margin: 0 auto;
-    display: grid;
     box-sizing: border-box;
 `;
 const centerChartStyle = css`
     margin: auto;
+    overflow: auto;
+    flex: 1 1 0;

```

Comment:
Remove: centerChartStyle is not set to `display: flex`, so the flex prop is useless here

Reply:
Done.